ee/maintained-apps/inputs/winget/scripts/microsoft_access_database_engine_2016_uninstall.ps1

```
@@ -0,0 +1,26 @@
+# The x86 and x64 redistributables share a DisplayName and differ only by
+# product code, so resolve the installed product from the x64 upgrade code.
+$upgradeCode = '{00160000-00D1-0000-1000-0000000FF1CE}'
+$timeoutSeconds = 300
+$successCodes = @(0, 3010, 1641)
+
+try {
+    $inst = New-Object -ComObject "WindowsInstaller.Installer"
+    $productCodes = @()
+    try { $productCodes = @($inst.RelatedProducts($upgradeCode)) } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)